### PR TITLE
Implementation of 2 channels display.

### PR DESCRIPTION
**Pull request by caballeto**
**Main work done:**
 - implemented functionality to display 2 channels
 - added `ftp` directory, where the files will be uploaded externally
 - hosted application on digital ocean Linux VM
 - runned gunicorn
 - implemented sample *uploader*, which can automatically upload data via ssh to server

**Notes:**
 - no dns set up for now
 - check site at http://206.81.21.147:5000/map (uploaded images are random)

### CODE REVIEW

File: directdemod/constants.py [hunk 1]
```diff
@@ -43,6 +44,12 @@
 MAP_TEMPLATE = MODULE_PATH + "/misc/map.html"
 GLOBE_TEMPLATE = MODULE_PATH + "/misc/globe.html"
 
+## SSH settings
```
Comment (7andahalf) [inline directdemod/constants.py:47]: What is this SSH for?

File: directdemod/server/conf.json [hunk 2]
```diff
@@ -1 +1 @@
-{"update_rate": 20, "counter": 4, "1": "1", "2": "2019-06-26_08:07:33_2019-06-26_08:07:53", "3": "2019-06-26_08:07:53_2019-06-26_08:08:13"}
\ No newline at end of file
+{"update_rate": 20, "counter": 4, "1": "1", "2": "2019-07-27_10:55:02_2019-07-27_10:55:22", "3": "2019-07-29_09:16:12_2019-07-29_09:16:32"}
```
Comment (7andahalf) [inline directdemod/server/conf.json:1]: I understand the "update_rate" parameter. What do the rest do?

**Review by 7andahalf — APPROVED** [review]
Great! Thanks!

File: directdemod/constants.py (hunk 1 above)
Comment (caballeto) [reply directdemod/constants.py:47]: These are special settings for ssh connection. The SSH connection is used to transmit data from the processing server to the webserver. There are two ways to upload images to the webserver:
1. Upload via a web site.
2. Upload via ssh.

Comment (caballeto) [reply directdemod/constants.py:47]: So basically there are two use cases:
1. Set of recordings stations transmit data to the processing server, that processes data and then sends it to the webserver via ssh.
2. Simply upload data via a web browser.

File: directdemod/server/conf.json (hunk 2 above)
Comment (caballeto) [reply directdemod/server/conf.json:1]: The counter is the parameter for a number of intervals (number of days). Next, are the pairs: day id and the corresponding time interval. So when it is chosen, the corresponding time label will be shown also.

**Pull request merged**